backend/app2/api/routes/assistant_streaming.py

```
@@ -58,7 +58,7 @@ class RequestCreationResponse(BaseModel):
 async def create_assistant_request(
     request: AssistantRequestModel,
     background_tasks: BackgroundTasks,
-    user: UserProfile = Depends(get_current_user)
+    user: Dict[str, Any] = Depends(get_current_user)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)